ibis/pyspark/tests/test_basic.py

```diff
@@ -163,6 +163,25 @@ def test_filter(client, filter_fn, expected_fn):
     tm.assert_frame_equal(result.toPandas(), expected.toPandas())
 
 
+@pytest.mark.parametrize('distinct', [False, True])
+def test_union(client, distinct):
```

Comment:
Do we have union tests under tests/all?

Reply:
I think I can move this test to tests/all actually

Reply:
I created a new `union` test in tests/all and removed the `union` test in the PySpark backend I was originally going to add. I also removed the existing basic `union` tests in the Pandas and Postgres backends